airflow/sensors/time_delta.py

```diff
@@ -28,6 +28,12 @@ class TimeDeltaSensor(BaseSensorOperator):
     Waits for a timedelta after the run's data interval.
 
     :param delta: time length to wait after the data interval before succeeding.
+
+    .. seealso::
+        For more information on how to use this sensor, take a look at the guide:
```

Comment:
Very minor thing: I see sometimes you wrote _"For more information on how to use this **sensor**"_ and sometimes you wrote _"For more information on how to use this **operator**"_. Does it make sense to you to make it more consistent?

Reply:
Yeeah. :). I think I missed it in a few places. This is verbatim copied from our "build-docs" warning when such clause is missing and "operator" is there, so I needed to manually change it. Obvioulsy missed it in a few places